src/applications/disability-benefits/686/config/form.js

```diff
@@ -18,6 +18,7 @@ import DependentField from '../components/DependentField';
 import createHouseholdMemberTitle from '../components/DisclosureTitle';
 import applicantDescription from '../../../../platform/forms/components/ApplicantDescription';
 import {
+  authorize,
```

Comment:
We will check for this on the dashboard page, and run for all gated forms that make this authorize helper available, to determine when to hide the loading indicator and reveal the saved forms/authorization errors.